libs/ui-lib/lib/ocm/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -24,15 +24,11 @@ import { DiscoveryImageModalButton } from './DiscoveryImageModal';
 import InformationAndAlerts from './InformationAndAlerts';
 import { OcmSwitchField } from '../ui/OcmFormFields';
 import { selectCurrentClusterPermissionsState } from '../../selectors';
```

Comment:
(Not for this line)
In `Host discovery`, we are using `useClusterSupportedPlatforms` to see if we need to patch the platformType (via `HostDiscoveryService.setPlatform`).
Is this still something we need?

Reply:
I think is not needed. @jkilzi I've removed it